docs/guides/deploy-guide/bootstrap.md

```diff
@@ -102,53 +152,56 @@ the nodes are already ready for use.
    * System is currently rebooting and is not yet accessible via network.
 
      ```
-     fatal: [net003]: UNREACHABLE! => {"changed": false, "msg": "Connection timed
-     out.", "unreachable": true}``
+fatal: [net003]: UNREACHABLE! =>
+{"changed": false, "msg": "Connection timed out.", "unreachable": true}
      ```
 
    * System has already been rebooted and is not accessible via the network.
 
      ```
-     fatal: [net003]: UNREACHABLE! => {"changed": false, "msg": "EOF on stream;
-     last 100 lines received:\nssh: connect to host 10.15.0.33 port 22: No route
-     to host\r", "unreachable": true}
+fatal: [net003]: UNREACHABLE! =>
+{"changed": false, "msg": "EOF on stream; last 100 lines received:\nssh: connect to host 10.15.0.33 port 22: No route to host\r", "unreachable": true}
      ```
 
-7. Refresh facts.
+## Bootstrap the nodes
+
+1. [Refresh facts](../operations-guide/manager/facts).
 
    ```
    osism apply facts
    ```
 
-8. Bootstrap.
+2. Bootstrap nodes
 
    ```
-   osism apply bootstrap
+   osism apply bootstrap -l 'all:!manager'
```

Comment:
No. That's wrong. Run the bootstrap for all nodes, include the manager node.

Reply:
My reasoning was that at this point of the installation the "bootstrap" play was already executed as part of the manager deployment and is therefore rather unnecessary here. But basically it shouldn't do any harm. I have therefore now removed the restriction.